pep-0698.rst

```diff
@@ -147,16 +147,15 @@ Many popular programming languages support override checks. For example:
 Runtime Override Checks in Python
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Today, there is an `Overrides library <https://pypi.org/project/overrides/>`_
-that provides decorators ``@overrides`` (sic) and ``@final`` and will enforce
-them at runtime.
+Today, there is an |overrides-library|_ that provides decorators
+``@overrides`` (sic) and ``@final`` and will enforce them at runtime.
```

Comment:
```suggestion
Today, there is an `Overrides library <https://pypi.org/project/overrides/>`__
that provides decorators ``@overrides`` [sic] and ``@final``
and will enforce them at runtime.
```

At least IMO, it's major overkill and a rather ugly and complicated hack to use two layers of substitutions here, just all to get ``overrides`` to display as a literal. There's really no practical need to put the proper name of a project in literals, unless it is being referred to in an actual literal context like a CLI command, Python code, string value, etc; otherwise, it's just visual noise, IMO, that dilutes the semantic meaning of literals in the source and the output. And nor does the project itself use this convention in it's own [Readme](https://github.com/mkorpela/overrides) (where it uses "overrides") or [Releases page](https://github.com/mkorpela/overrides/releases)_ (where it uses "Overrides").

Reply:
I have no objections to reverting this -- the project name was formatted as code-literal elsewhere in the PEP, and that's my personal preference; thus the suggestion.

(I completely agree that the double substitution is bulky, and I was expecting pushback on those grounds. But, AFAIK it's the only way to achieve this in reST.)

If we revert here, then I'd advocate for converting formatting of *all* references to the project to whatever form, 'overrides' or 'Overrides'.

Reply:
Ah, I see—I should have checked that, sorry.

> If we revert here, then I'd advocate for converting formatting of all references to the project to whatever form, 'overrides' or 'Overrides'.

That would be my suggestion as well, as personally I'm not really a big fan of formatting project names in literals everywhere without a clear semantically-appropriate reason, as it just adds noise and complexity in both the source and the output, and leads to cases like this. Though, even if not, I'd still say it's not worth all this just for the link, viz. "A foolish consistency..." However, it's ultimately up to @stroxler as PEP author.

Reply:
I think we could drop the formatting of the library name everywhere, and just format the decorator names. The library name itself isn't really a code snippet.